packages/manager/src/features/Kubernetes/kubeUtils.ts

```diff
@@ -179,7 +179,7 @@ export const getLatestVersion = (
   versions: { label: string; value: string }[]
 ): { label: string; value: string } => {
   const sortedVersions = versions.sort((a, b) => {
-    return sortByVersion(a.value, b.value, 'asc');
+    return sortByTieredVersion(a.value, b.value, 'asc');
```

Comment:
```suggestion
    return sortByTieredVersion(a.value, b.value, 'desc');
```
Shouldn't this be 'desc' instead of 'asc', since we are expecting the versions to be sorted by returning the latest version first?

Reply:
This should be the order in which to display the sorted versions (output), not the order that the data is coming from the API (input). And we want the sorted versions to be in asc order so that we can pop the latest version off the bottom of the array.

Thanks for the point of clarification - I updated the JSDoc comment for the `sort` param in both sortBy functions to hopefully make its use more clear.